test/bedrock_Azure_mc_test.go

```diff
@@ -61,6 +64,22 @@ func TestIT_Bedrock_AzureMC_Test(t *testing.T) {
 	azureCommonInfraFolder := "../cluster/test-temp-envs/azure-common-infra-" + k8sName
 	copy.Copy("../cluster/environments/azure-common-infra", azureCommonInfraFolder)
 
+        //Create the common resource group
+        cmd0 := exec.Command("az", "login", "--service-principal", "-u", clientid, "-p", clientsecret, "--tenant", tenantid)
```

Comment:
In the future, we may be able to use the Terratest module to create the RG.  - https://github.com/gruntwork-io/terratest/pull/333

Reply:
That's a good fix.  Let's file an issue on it.

Reply:
Issue for reference - https://github.com/microsoft/bedrock/issues/553